app/assets/stylesheets/common.scss

```diff
@@ -995,25 +995,25 @@ img.trace_image {
 .apple-logo rect { fill: black; }
 .apple-logo path { fill: white; }
 
-@media (prefers-color-scheme: dark) {
+@include color-mode(dark) {
   .apple-logo rect { fill: white; }
   .apple-logo path { fill: black; }
 }
 
-.github-logo rect { fill: white; }
-.github-logo path { fill: black; }
+.github-logo rect { fill: black; }
+.github-logo path { fill: white; }
 
-@media (prefers-color-scheme: dark) {
-  .github-logo rect { fill: black; }
-  .github-logo path { fill: white; }
+@include color-mode(dark) {
+  .github-logo rect { fill: white; }
+  .github-logo path { fill: #1b1f24; }
 }
 
 /* Rules for contextmenu */
 
 #map-context-menu {
   .dropdown-item {
     &:hover, &:active {
-      background-color: $gray-200;
+      @extend :focus;
```

Comment:
How about we just don't override the hover and active states at all, and let bootstrap deal with them? If we leave it alone, it works fine in both dark and light modes, as far as I can tell.

```diff
 #map-context-menu {
-  .dropdown-item {
-    &:hover, &:active {
-      @extend :focus;
-    }
-  }
   &.cm_dropdown {
```